src/pages/MovieDetail.jsx

```diff
@@ -0,0 +1,104 @@
+import React,{useEffect} from 'react'
+import { useParams } from 'react-router';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMovieAction,addMovieWishlistAction } from '../store/actions/moviesAction';
+import movieConfig from '../config/themoviedb';
+import styled from 'styled-components';
+
+const Genres = styled.div`
+    display:inline-block;
+    background-color:#CCCCCC;
+    margin-right:10px;
+    padding:5px;
+    border-radius:5px;
+`;
+
+const Gallery = styled.img`
+    max-width:250px;
+    margin-bottom:10px;
+    margin-right:5px;
+    margin-left:5px;
+`;
+
+const Wishlist = styled.i`
+
+`;
```

Comment:
In general, it is better to not include code that has no functionality at the moment. In this case, I would remove this component and simply use an `i` tag in the component.